src/components/ViewInstance/MenuSection/NewInstanceRequestBtn.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { injectIntl } from 'react-intl';
+
+import { IfPermission } from '@folio/stripes/core';
+import {
+  Button,
+  Icon,
+} from '@folio/stripes/components';
+
+import { layers } from '../../../constants';
+
+export const getInstanceRequestLink = (instanceId) => `/requests?instanceId=${instanceId}&layer=${layers.CREATE}`;
+
+const NewInstanceRequestBtn = ({
+  isTlrEnabled,
+  instanceId,
+  intl: {
+    formatMessage,
+  },
```

Comment:
You can also do `const intl = useIntl();` to get at the `intl` object via a hook instead of wrapping the exported component with `injectIntl`. You don't need to change this here, but I think it's considered the more idiomatic way to access it in a functional component.

Reply:
Good point, thanks! Done.